Pkgfile

```diff
@@ -114,14 +114,14 @@ vars:
   liburcu_sha512: 7297e51012f4c44ee27c0e18ed9d87bf24be34db68a5398394c1e683a045bb561cf74aa913398404c0ed5cb8011af728ea12947717fa5f27627e5ca78e63a40f
 
   # renovate: datasource=git-tags depName=git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git
-  linux_firmware_version: 20240610
-  linux_firmware_sha256: fe7687f7264ef1193704824c884be515cd385c15303962cb85c929d40c6cd5b9
-  linux_firmware_sha512: fde9a5acb3407335227e7a2184335b21fc2c740503c6ff287f60ad10ae49ee86efaa07da1483a7075668820d16a040af4f87918ff5ff21b816fc42a98a1bfc0c
+  linux_firmware_version: 20240709
```

Comment:
the version tag should be updated in `extensions` as well as we pull it in there (just a reminder)

Reply:
Should be covered here https://github.com/siderolabs/extensions/pull/443